style.css

```diff
@@ -282,7 +392,9 @@
     transition: background-color 0.5s, width 0.5s, color 0.5s;
   }
 
-  .button-1:hover {
+  .button-1:hover,
+  .button-2:hover,
+  .button-3:hover {
     box-shadow: 0 8px 16px;
     border-color: rgba(255, 97, 99, 0.24);
     background-color: aqua;
```

Comment:
- [ ] To match the original design, I noticed your hover styles are not the same as the original design styles. Kindly take off the border from the buttons, and adjust the background color for the buttons on hover to match the design and not the aqua color you're having. 

**Your design**

<img width="327" alt="Screenshot 2022-12-23 at 3 07 36 AM" src="https://user-images.githubusercontent.com/50941074/209263523-25cc3ca7-1a54-48e0-826a-066b049af15e.png">

**The original design**

Kindly modify your button styles 

<img width="236" alt="Screenshot 2022-12-23 at 3 14 10 AM" src="https://user-images.githubusercontent.com/50941074/209263814-61e71deb-48fe-46cd-8d64-0ede7e51c8cc.png">

Please use these colors from the template to match your design with the original design given on the hover of a button. In your case, you'll use the green color. 👍🏽

<img width="509" alt="Screenshot 2022-12-23 at 3 12 55 AM" src="https://user-images.githubusercontent.com/50941074/209263697-5a06fb3b-2b46-412f-96e5-9ec5fe40b31d.png">

Reply:
Thank you for the helpful feedback. I have made the requested changes and set the button borders and outlines to none. I've also altered the hover styling of the buttons to match the figma template.